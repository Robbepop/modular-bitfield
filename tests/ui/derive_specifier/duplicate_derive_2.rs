use modular_bitfield::prelude::*;

#[bitfield]
#[derive(BitfieldSpecifier)] #[derive(BitfieldSpecifier)]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
