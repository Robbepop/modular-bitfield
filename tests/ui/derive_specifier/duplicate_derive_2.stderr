error: encountered duplicate `#[derive(BitfieldSpecifier)]` parameter
 --> tests/ui/derive_specifier/duplicate_derive_2.rs:4:39
  |
4 | #[derive(BitfieldSpecifier)] #[derive(BitfieldSpecifier)]
  |                                       ^^^^^^^^^^^^^^^^^

error: previous `#[derive(BitfieldSpecifier)]` parameter here
 --> tests/ui/derive_specifier/duplicate_derive_2.rs:4:10
  |
4 | #[derive(BitfieldSpecifier)] #[derive(BitfieldSpecifier)]
  |          ^^^^^^^^^^^^^^^^^
